sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1152,6 +1152,16 @@ def __init__(self, eps=1e-3, n_alphas=100, alphas=None, fit_intercept=True,
         self.random_state = random_state
         self.selection = selection
 
+    @abstractmethod
+    def _get_model(self):
+        """Model to be fitted after the best alpha has been determined."""
+        return
```

Comment:
use `pass` instead maybe
or raise a NonImplementedError
We can also just not have these in the ABC

Reply:
@agramfort which one of these possibilities do you prefer ?

Reply:
With the docstring, we do not need anything in the body.

Reply:
@thomasjpfan  Removed return. I personally think having the methods in the ABC is clearer (my 2 cents)

Reply:
I think it's usual coding convetion for `@abstractmethod` to have an empty body (just a docstring).